src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Employee/Blocks/change_password.html.twig

```diff
@@ -0,0 +1,97 @@
+{% set newPasswordFirstVars = change_password.new_password.children.first.vars|merge(new_password.first_options|default({})) %}
+{% set newPasswordSecondVars = change_password.new_password.children.second.vars|merge(new_password.second_options|default({})) %}
+<div class="form-group row">
+  <label class="form-control-label">
+    {{ 'Password'|trans({}, 'Admin.Global') }}
+  </label>
+  <div class="col-sm">
+    {# "Change password" button #}
+    {{ form_widget(change_password.change_password_button, {
+      label: 'Change password...'|trans({}, 'messages'),
+      attr: {
+        class: 'btn-outline-secondary js-change-password',
+      }
+    }) }}
+
+    <div class="card card-body js-change-password-block d-none">
+      {# Current password input #}
+      {{ ps.form_group_row(change_password.old_password, oldPasswordVars, {
+        label: 'Current password'|trans({}, 'messages'),
+        required: true,
+      }) }}
+
+      {# New password first input #}
+      <div class="form-group row">
+        {{ ps.label_with_help('New password'|trans({}, 'messages'), passwordHelpMessage, '', change_password.new_password.children.first.vars.id, true) }}
+        <div class="col-sm">
+          {{ ps.form_widget_with_error(change_password.new_password.children.first, change_password.new_password.children.first.vars) }}
+        </div>
+      </div>
+
+      {# New password second input (confirmation) #}
+      {# Empty help text needed to render the text container for validation feedback messages #}
+      {{ ps.form_group_row(change_password.new_password.children.second, newPasswordSecondVars, {
+        label: 'Confirm password'|trans({}, 'messages'),
+        help: '',
+        required: true,
+      }) }}
+
+      <div class="form-group row">
+        <label class="form-control-label"></label>
+        <div class="col-sm">
+          {{ form_widget(change_password.generated_password) }}
+        </div>
+        <div class="col-sm">
+          {{ form_widget(change_password.generate_password_button, {
+            label: 'Generate password'|trans({}, 'messages'),
+            attr: {
+              class: 'btn-outline-secondary',
+            }
+          }) }}
+        </div>
+      </div>
+
+      {{ ps.form_group_row(change_password.cancel_button, {
+        label: 'Cancel'|trans({}, 'Admin.Actions'),
+        attr: {
+          class: 'btn-outline-secondary js-change-password-cancel',
+        }
+      }) }}
+
+      {# Password strength feedback messages - used in JS #}
+      <div class="js-password-strength-feedback d-none">
+        <span class="strength-low">{{ 'Invalid'|trans({}, 'messages') }}</span>
+        <span class="strength-medium">{{ 'Okay'|trans({}, 'messages') }}</span>
+        <span class="strength-high">{{ 'Good'|trans({}, 'messages') }}</span>
+        <span class="strength-extreme">{{ 'Fabulous'|trans({}, 'messages') }}</span>
```

Comment:
We have the same issue here. @LouiseBonnard Any idea why there are so many wordings in the "messages" category here? Maybe there is a reason I am missing.

Reply:
Not at all! As explained [here](https://docs.google.com/presentation/d/18fCfwr4erxwflhq5vT8TbNOCnYOBj5d3OIJOig4Zxso/edit#slide=id.g711cd0b5aa_0_118), all the strings that have not been assigned a domain so far appear in the _messages_ domain. It is not the expected behavior, you're right, and they are inherited from the previous versions so they should get a more accurate domain each time we stumble upon them during wording reviews. ;-)

Reply:
That's what I thought! :) Thank you Louise!